### PR TITLE
Stylelint Update

### PR DESCRIPTION
**Pull request by Ne-Ne**
Update package.json and increment revision version number.

### CODE REVIEW

**Comment by Ne-Ne** [comment]
Are you happy with these changes @olegskl?

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "gulp-stylelint",
-  "version": "3.7.0",
+  "version": "3.7.1",
```
Comment (olegskl) [inline package.json]: There's no need to bump the lib version yourself (it should be a minor, not patch anyway). It will bumped before release.

Comment (Ne-Ne) [inline package.json]: Do you know when release is due?

Comment (olegskl) [inline package.json]: Today.

**Comment by olegskl** [comment]
Closed in favor of 5a9d6ea9564f0430aba8c560c74bd902b7597e09

**Pull request closed without merging**